rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -448,3 +448,10 @@ maas_excluded_checks: []
 # This is necessary until LP #1537117 is implemented
 openrc_os_endpoint_type: internalURL
 openrc_insecure: "{{ (keystone_service_adminuri_insecure | bool or keystone_service_internaluri_insecure | bool) | default(false) }}"
+# Name of the virtual env to deploy into (maas)
+maas_venv_tag: untagged
```

Comment:
Not sure it's the worst either. That said, I'd think this should coincide with RPCO's release number, if we store that as a var (which I don't remember if we do).